dhall-nix/src/Dhall/Nix.hs

```diff
@@ -208,13 +209,13 @@ Right x: y: x + y
 dhallToNix :: Expr s Void -> Either CompileError (Fix NExprF)
 dhallToNix e = loop (Dhall.Core.normalize e)
   where
-    loop (Const _) = return (Fix (NSet []))
+    loop (Const _) = return (Fix (NSet NNonRecursive []))
```

Comment:
So my understanding is that these empty sets are a bit like a default value for things that I can't translate. Should I make  a name for this to avoid repetition? Something like

```haskell
stub = Fix (NSet NNonRecursive [])
```

?

Reply:
Yeah, currently all types are translated to `{}` in Nix.  You can define a reusable term to reduce repetition

Reply:
I went with `untranslatable` for the name.